db/migrate/20180920171008_change_repository_password_to_text.rb

```diff
@@ -0,0 +1,17 @@
+class ChangeRepositoryPasswordToText < ActiveRecord::Migration[5.2]
+  def up
+    change_column :katello_repositories, :upstream_password, :text
+  end
+
+  def down
+    add_column :katello_repositories, :temp_upstream_password, :string, :limit => 1024
+
+    Katello::Repository.find_each do |repo|
```

Comment:
I think you may need a fake class here - [example here](https://github.com/Katello/katello/blob/master/db/migrate/20171010172724_add_docker_manifest_list.rb#L42)

@jlsherrill can confirm or deny as he is the one who explained the reasoning behind this to me and I'm struggling to remember :thinking:

Reply:
I don't think its necessary here because he is using update_column which ignores validations and callbacks.  The other methods he is calling are just column-based attributes.